src/main/java/com/kodilla/ecommercee/controller/UserController.java

```diff
@@ -31,31 +28,22 @@ public UserDto getUser(@RequestParam Long userId) throws UserNotFoundException {
         return userMapper.mapToUserDto(service.getUser(userId).orElseThrow(UserNotFoundException::new));
     }
 
-    @RequestMapping(method = RequestMethod.DELETE, value = "deleteUser")
-    public void deleteUser(@RequestParam Long userId) {
-            service.delete(userId);
-    }
 
-    @RequestMapping(method = RequestMethod.PUT, value = "updateUser")
-    public UserDto updateUser(@RequestBody UserDto userDto) {
+    @RequestMapping(method = RequestMethod.PUT, value = "updateUserDto")
+    public UserDto updateUserDto(@RequestBody UserDto userDto) {
```

Comment:
Nie do końca rozumiem jakie jest zastosowanie tej metody

Reply:
W createUser używasz post w update aktualizujesz przez put. Metody w sumie sa dwie do aktualizacji Usera poprzez UserDto oraz przez Encje User. W założeniu po to gdyby trzeba bylo cos zmienić np status z 1 na 0 aby dezaktywowac użykownika. Z komentarzy powyzej stworze osobna metode do dezaktywowania użytkownika (poprzez zmiane statusu) ale zakłądam że przywrócenie bedzie sie odbywać poprzez metode update.

Reply:
Ok, to chyba rozumiem zamysł :) Z tym, że wydawało mi się, że po to używamy obiektów dto żeby odseparować tą warstwę wewnętrzną aplikacji od zewnętrznej i nie pozwalać na zmiany w tej wewnętrznej warstwie bezpośrednio przez input z zewnątrz, tylko żeby jakoś to "kontrolować". Tak żeby zewnętrznie nie dało się operować na bazie danych, co ja w jakiś sposób zabezpiecza. Nie wiem czy dobrze udało mi się wyjaśnić o co chodzi :D